test/Scripts.Tests/package-release.zip.snapshot

```diff
@@ -202,13 +202,9 @@ Plugins/Linux/Sentry/libsentry.so
 Plugins/Linux/Sentry/libsentry.so.meta
 Plugins/Android/proguard-sentry-unity.pro
 Plugins/Android/proguard-sentry-unity.pro.meta
-Plugins/Android/Sentry.meta
-Plugins/Android/Sentry/sentry-android-core-release.aar
-Plugins/Android/Sentry/sentry-android-core-release.aar.meta
-Plugins/Android/Sentry/sentry-android-ndk-release.aar
-Plugins/Android/Sentry/sentry-android-ndk-release.aar.meta
-Plugins/Android/Sentry/sentry.jar
-Plugins/Android/Sentry/sentry.jar.meta
+Plugins/Android/Sentry~/sentry-android-core-release.aar
+Plugins/Android/Sentry~/sentry-android-ndk-release.aar
+Plugins/Android/Sentry~/sentry.jar
```

Comment:
where are the .meta files?

Reply:
We don't need/have meta files for those anymore. Appending `~` to a directory makes Unity ignore it and we're copying and setting up the Android SDK ourselves.